shared/js/project.js

```diff
@@ -144,7 +141,7 @@ var projects = module.exports = {
    *   exceed our threshold
    */
   exceedsAbuseThreshold: function () {
-    return !!(current && current.abuseScore && current.abuseScore >= ABUSE_THRESHOLD);
+    return !!(currentAbuseScore >= ABUSE_THRESHOLD);
```

Comment:
Hmm this is failing jslint for me: 
```
[CDO]/shared/js/project.js: line 144, col 13, Confusing use of '!'.
```
RubyMine suggests `return currentAbuseScore >= ABUSE_THRESHOLD;`